packages/app-config/scripts/package-subtree-local.sh

```
@@ -0,0 +1,1201 @@
+      echo ""
+      echo "This means the branch in the local repository doesn't contain the commit"
+      echo "that the monorepo was last synced to. Possible causes:"
+      echo "  * The local branch was rebased or reset"
+      echo "  * The local repo is pointing to a different fork"
+      echo "  * The branch doesn't share history with the upstream"
+      clean_exit 1 "" true
+    fi
+  fi
+fi
+
+if [ ! -d "$TARGET_DIR" ] && ([ -z "$CURRENT_COMMIT" ] || [ "$CURRENT_COMMIT" = "null" ] || [ "$CURRENT_COMMIT" = "" ]); then
```

Shellcheck (0.11.0)

[style] Use { ..; } instead of (..) to avoid subshell overhead.

(SC2235)